pkgs/development/python-modules/httpbin/default.nix

```diff
@@ -62,6 +62,11 @@ buildPythonPackage rec {
     "test_redirect_to_post"
     "test_relative_redirect_n_equals_to_1"
     "test_relative_redirect_n_higher_than_1"
+    # break with new werkzeug
+    # disable until PSF fork catches up: https://github.com/psf/httpbin
+    "test_base64"
+    "test_digest_auth"
+    "test_digest_auth_wrong_pass"
```

Comment:
Would you mind trying if these two patches fix these test issues?

https://github.com/psf/httpbin/commit/b4e9808a628d1460346ec8790d810586f03bb4ba.patch

https://github.com/psf/httpbin/commit/7facfd8ea1983a4dd8e759d11155e81a682c16a1.patch

Reply:
Indeed, it worked: https://github.com/NixOS/nixpkgs/compare/77adbeb69e1da83bc8f87e05667345fbbc00aa77..c17fc82201157b2f00e5520052ac8de30062f78c